nav2_docking/opennav_docking/src/docking_server.cpp

```diff
@@ -59,6 +61,17 @@ DockingServer::on_configure(const rclcpp_lifecycle::State & state)
   get_parameter("fixed_frame", fixed_frame_);
   get_parameter("dock_backwards", dock_backwards_);
   get_parameter("dock_prestaging_tolerance", dock_prestaging_tolerance_);
+  get_parameter("initial_rotation", initial_rotation_);
+  get_parameter("backward_blind", backward_blind_);
+  if(backward_blind_ && !dock_backwards_){
+    RCLCPP_ERROR(get_logger(), "backward_blind is enabled when dock_backwards is disabled.");
+    return nav2_util::CallbackReturn::FAILURE;
+  } else{
+    // If you have backward_blind and dock_backward then
+    // we know we need to do the initial rotation to go from forward to reverse
+    // before doing the rest of the procedure. The initial_rotation would thus always be true.
+    initial_rotation_ = true;
```

Comment:
Does `initial_rotation_` even need to be a parameter? If `backward_blind_ = true` don't we always do the rotation?

Reply:
It also seems unused in the code itself :-)